packages/l/lime3ds/pspec_x86_64.xml

```diff
@@ -0,0 +1,62 @@
+<PISI>
+    <Source>
+        <Name>lime3ds</Name>
+        <Homepage>https://lime3ds.github.io/</Homepage>
+        <Packager>
+            <Name>Muhammad Alfi Syahrin</Name>
+            <Email>malfisya.dev@hotmail.com</Email>
+        </Packager>
+        <License>GPL-2.0-or-later</License>
+        <PartOf>games.emulator</PartOf>
+        <Summary xml:lang="en">A Nintendo 3DS emulator based on Citra</Summary>
+        <Description xml:lang="en">Lime3DS is a project which aims to revive and continue work on Citra, a popular open-source Nintendo 3DS emulator which ceased development.
+</Description>
+        <Archive type="binary" sha1sum="79eb0752a961b8e0d15c77d298c97498fbc89c5a">https://sources.getsol.us/README.Solus</Archive>
+    </Source>
+    <Package>
+        <Name>lime3ds</Name>
+        <Summary xml:lang="en">A Nintendo 3DS emulator based on Citra</Summary>
+        <Description xml:lang="en">Lime3DS is a project which aims to revive and continue work on Citra, a popular open-source Nintendo 3DS emulator which ceased development.
+</Description>
+        <PartOf>games.emulator</PartOf>
+        <Files>
+            <Path fileType="executable">/usr/bin/lime3ds-cli</Path>
+            <Path fileType="executable">/usr/bin/lime3ds-gui</Path>
+            <Path fileType="executable">/usr/bin/lime3ds-room</Path>
+            <Path fileType="data">/usr/share/applications/lime3ds-gui.desktop</Path>
+            <Path fileType="data">/usr/share/icons/hicolor/scalable/apps/lime.svg</Path>
+            <Path fileType="data">/usr/share/mime/packages/lime.xml</Path>
+        </Files>
+    </Package>
+    <Package>
+        <Name>lime3ds-devel</Name>
+        <Summary xml:lang="en">Development files for lime3ds</Summary>
+        <Description xml:lang="en">Lime3DS is a project which aims to revive and continue work on Citra, a popular open-source Nintendo 3DS emulator which ceased development.
+</Description>
+        <PartOf>programming.devel</PartOf>
+        <RuntimeDependencies>
+            <Dependency release="1">lime3ds</Dependency>
+        </RuntimeDependencies>
+        <Files>
+            <Path fileType="header">/usr/include/enet/callbacks.h</Path>
+            <Path fileType="header">/usr/include/enet/enet.h</Path>
+            <Path fileType="header">/usr/include/enet/list.h</Path>
+            <Path fileType="header">/usr/include/enet/protocol.h</Path>
+            <Path fileType="header">/usr/include/enet/time.h</Path>
+            <Path fileType="header">/usr/include/enet/types.h</Path>
+            <Path fileType="header">/usr/include/enet/unix.h</Path>
+            <Path fileType="header">/usr/include/enet/utility.h</Path>
+            <Path fileType="header">/usr/include/enet/win32.h</Path>
+            <Path fileType="library">/usr/lib64/libenet.a</Path>
+        </Files>
```

Comment:
I am quite sure how to disable this. Citra doesn't have `-devel` packages and those files is certainly not lime's

Reply:
Used system `libenet` solves this